### PR TITLE
DOCSP-26498 - Standardize MongoDB Server Naming

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-26498
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/docsp-26498-mongodb-caps/

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [X] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/write-operations/change-a-document.txt [hunk 1]
```diff
@@ -128,7 +128,7 @@ documents match.
 
     The ``updateOne()`` and ``updateMany()`` methods cannot make changes
     to a document that violate unique index constraints on the
-    collection. See the MongoDB server manual for more information on
+    collection. See the MongoDB Server manual for more information on
     :manual:`unique indexes </core/index-unique/>`.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/crud/write-operations/change-a-document.txt:132]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

```suggestion
    collection. For more information about unique index constraints, see 
    :manual:`unique indexes </core/index-unique/>` in the {+mdb-server+} manual. 
```

File: source/fundamentals/crud/write-operations/change-a-document.txt [hunk 2]
```diff
@@ -206,4 +206,4 @@ the ``replaceOne()`` method, it replaces the first result.
 
     The ``replaceOne()`` method cannot make changes to a document that
     violate unique index constraints on the collection. See the MongoDB
-    server manual for more information on :manual:`unique indexes </core/index-unique/>`.
+    Server manual for more information on :manual:`unique indexes </core/index-unique/>`.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/crud/write-operations/change-a-document.txt:209]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt [hunk 3]
```diff
@@ -165,7 +165,7 @@ The following is an example of a query that would be covered by the index create
 
 .. common-content-begin
 
-See the MongoDB server manual section on :manual:`single field indexes </core/index-single>` for more information.
+See the MongoDB Server manual section on :manual:`single field indexes </core/index-single>` for more information.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:168]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt [hunk 4]
```diff
@@ -200,7 +200,7 @@ The following is an example of a query that would be covered by the index create
 
 .. common-content-begin
 
-See the MongoDB server manual section on :manual:`Compound indexes </core/index-compound>` for more information.
+See the MongoDB Server manual section on :manual:`Compound indexes </core/index-compound>` for more information.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:203]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt [hunk 5]
```diff
@@ -306,7 +306,7 @@ the geospatial data for inclusion, intersection, and proximity. For more informa
 :manual:`Geospatial Queries </geospatial-queries/>`.
 
 To create a ``2dsphere`` index, you must specify a field that contains only **GeoJSON objects**. For more details on this
-type, see the MongoDB server manual page on :manual:`GeoJSON objects </reference/geojson>`.
+type, see the MongoDB Server manual page on :manual:`GeoJSON objects </reference/geojson>`.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:309]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt [hunk 6]
```diff
@@ -361,7 +361,7 @@ The following is an example of a geospatial query using the "location.geo" index
 
 MongoDB also supports ``2d`` indexes for calculating distances on a Euclidean plane and for working with the "legacy
 coordinate pairs" syntax used in MongoDB 2.2 and earlier. See the :manual:`Geospatial Queries page </geospatial-queries>`
-in the MongoDB server manual for more information.
+in the MongoDB Server manual for more information.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:364]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt [hunk 7]
```diff
@@ -393,7 +393,7 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
       E11000 duplicate key error index
 
-Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
+Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB Server manual for more information.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:396]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt [hunk 8]
```diff
@@ -418,7 +418,7 @@ the ``vendors`` collection:
 
    database.createCollection("vendors", createCollectionOptions);
 
-See the MongoDB server manual sections for more information:
+See the MongoDB Server manual sections for more information:
```
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:421]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/bulkWrite.txt [hunk 9]
```diff
@@ -37,7 +37,7 @@ The ``bulkWrite()`` method accepts the following parameters:
 
 .. note::
 
-   Retryable writes run on MongoDB server versions 3.6 or later in bulk
+   Retryable writes run on MongoDB Server versions 3.6 or later in bulk
```
Comment (DBirtolo-mdb) [inline source/usage-examples/bulkWrite.txt:40]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/command.txt [hunk 10]
```diff
@@ -6,7 +6,7 @@ Run a Command
 
 You can run all raw database operations using the
 ``MongoDatabase.runCommand()`` method. A raw database operation is a
-command you can execute directly on the MongoDB server CLI. These
+command you can execute directly on the MongoDB Server CLI. These
```
Comment (DBirtolo-mdb) [inline source/usage-examples/command.txt:9]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/count.txt [hunk 11]
```diff
@@ -43,10 +43,10 @@ When you call the ``countDocuments()`` method, you can optionally pass a
 .. important:: Stable API V1 and MongoDB Server Issue
 
    If you are using the Stable API ``V1`` with the "strict" option and a 
-   MongoDB server version between 5.0.0 and 5.0.8 inclusive, method calls to
+   MongoDB Server version between 5.0.0 and 5.0.8 inclusive, method calls to
```
Comment (DBirtolo-mdb) [inline source/usage-examples/count.txt:46]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/count.txt [hunk 12]
```diff
@@ -43,10 +43,10 @@ When you call the ``countDocuments()`` method, you can optionally pass a
...
    ``estimatedDocumentCount()`` may error due to a server bug. 
 
-   Upgrade to MongoDB server 5.0.9 or set the Stable API "strict" option to 
+   Upgrade to MongoDB Server 5.0.9 or set the Stable API "strict" option to
```
Comment (DBirtolo-mdb) [inline source/usage-examples/count.txt:49]: [question]

Is there a reason this is not {+mdb-server+}?

**Review by DBirtolo-mdb — CHANGES_REQUESTED** [review]
Nice job! Mainly two things (but there is at least one other one-off comment in there):

* Using the placeholder constant text
* Recommended link text updates

File: source/fundamentals/crud/write-operations/change-a-document.txt (hunk 1 above)
Comment (mongoKart) [reply source/fundamentals/crud/write-operations/change-a-document.txt:132]: Sure. Wasn't looking at things like that for this ticket, but happy to change.

File: source/fundamentals/indexes.txt (hunk 5 above)
Comment (mongoKart) [reply source/fundamentals/indexes.txt:309]: This looks okay to me...?

File: source/fundamentals/indexes.txt [hunk 13]
```diff
@@ -85,7 +85,7 @@ are arbitrary. MongoDB 4.2 introduced :manual:`wildcard indexes </core/index-wil
 Wildcard indexes are not designed to replace workload-based index planning.
 
 For more information on designing your data model and choosing indexes appropriate for your application, see the MongoDB
-server documentation on :manual:`Indexing Strategies </applications/indexes>` and
+Server documentation on :manual:`Indexing Strategies </applications/indexes>` and
```
[the remark this answers is not in the document]
Comment (DBirtolo-mdb) [reply source/fundamentals/indexes.txt:88]: Missed this one, but otherwise all good!

**Review by DBirtolo-mdb — APPROVED** [review]
LGTM

There was one instance where you didn't replace "MongoDB Server" with the constant (in source/fundamentals/indexes.txt), but that wasn't a critical change, so didn't want to hold up approval.

**Pull request merged**